### DISCUSSION TITLE
shapes

### DISCUSSION THREAD

**Discussion by DalilaBA** [flxzt/rnote#708] [Q&A] [answered]
hi I coudn't create shapes , like cercle or square...
![image_2023-06-12_205808171](https://github.com/flxzt/rnote/assets/10799976/b6e24101-611f-48c8-9ad8-4e7070333104)

**Comment by Heiseneberg-maybe** [comment, marked as answer]
Click on the line icon, a menu called Shape Builders will open. There you can choose the shapes you want to use.
To create a perfect circle or square you can activate the constraint 1:1 in the menu of the constraints(the last object on the sidebar, the one with a gear symbol) alternatively you can just draw a rectangle or ellipse and make it look like a square or a circle

**Comment by DalilaBA** [comment]
thank you very much for your help
